### PR TITLE
Linter fixes

### PR DESCRIPTION
**Pull request by martingartonft**
I only renamed stuff, so in theory nothing could possibly break, but I DIDN'T TEST IT.

### CODE REVIEW

**Comment by Nesze** [comment]
+1

File: notifications_push.go [hunk 1]
```diff
@@ -59,33 +59,33 @@ func main() {
...
+		http.HandleFunc("/__health", hc.healthcheck())
+		http.HandleFunc("/__gtg", hc.gtg)
 
 		go func() {
 			err := http.ListenAndServe(":8080", nil)
 			errorLogger.Println(err)
 		}()
 
-		notificationsApp.consumeMessages()
+		app.consumeMessages()
```
Comment (kovacshuni) [inline notifications_push.go]: app inside and app outside?

**Comment by kovacshuni** [comment]
+1

File: notifications_push.go (hunk 1 above)
Comment (martingartonft) [inline notifications_push.go]: This will work ok but I agree might be confusing because the variable name is shadowed.

**Comment by martingartonft** [comment]
As you guys are working on this still, do you mind picking this up and merging when it's convenient please?

**Comment by Nesze** [comment]
It's still in progress. But we can merge this right now.

**Pull request merged**